CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# Contributing
+
+Contributions to this repository will be very welcome, both in the form of issues or pull-requests 😊
+
+Before starting to work on a particular piece of code, please first open an issue to discuss the change before starting to work on it.
+
+Please note that Elastic does have a [Code of Conduct](https://www.elastic.co/community/codeofconduct).
+
+
+# Creating an issue
+
+We expect contributors to fill our issue templates when creating new issues. It's important that contributors use those templates appropriately because doing so helps us triage issues more quickly, discuss them more efficiently, and respond to the issue's participants sooner.
+
+For bugs, for example, we expect contributors to provide clear instructions on how to reproduce the bug they've found, and, if possible a screenshot or GIF illustrating the problem. We also appreciate it when contributors can provide information with regards to what caused the bug to occur, even though that's not necessary for reporting a bug.
```

Comment:
Super nit, just trying to pare off a couple of words.

```suggestion
For bugs, for example, we expect contributors to provide clear instructions on how to reproduce the bug they've found, and, if possible a screenshot or GIF illustrating the problem. We also appreciate it when contributors can provide information about the bug's cause, even though that's not necessary for reporting a bug.
```

Reply:
Great suggestion, I love it when we can make text more concise, thank you 💖